fast64_internal/f3d/f3d_gbi.py

```diff
@@ -23,6 +23,12 @@ class GfxListTag(enum.Enum):
     Draw = 3
 
 
+class GfxTag(enum.Enum):
+    NoTag = 0
+    TileScroll0 = 1
+    TileScroll1 = 2
```

Comment:
```suggestion
class GfxTag(enum.Enum):
    NoTag = enum.auto()
    TileScroll0 = enum.auto()
    TileScroll1 = enum.auto()
```

Thought on making this `enum.Flag` instead of an Enum? then a single macro can be tagged different things

e.g. `macro.tags |= GfxTag.TAG_FOO` `macro.tags & GfxTag.TAG_BAR`